animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -259,12 +264,39 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
         stepCos = cos(angleStep)
         stepSin = sin(angleStep)
 
+        if self.planeAxis == "XY":
+            for i in range(amount):
+                vector.x = iCos * radius
+                vector.y = iSin * radius
+                vector.z = 0
+
+                setTranslationMatrix(matrices.data + i, &vector)
+                setMatrixCustomZRotation(matrices.data + i, iCos, iSin)
+
+                rotateStep(&iCos, &iSin, stepCos, stepSin)
+
+            return matrices
+
+        elif self.planeAxis == "YZ":
```

Comment:
If you are going to use a guard condition, this should be an `if`.

Reply:
Changed.